botocore/exceptions.py

```diff
@@ -342,17 +342,30 @@ class UnsupportedSignatureVersionError(BotoCoreError):
 class ClientError(Exception):
     MSG_TEMPLATE = (
         'An error occurred ({error_code}) when calling the {operation_name} '
-        'operation: {error_message}')
+        'operation{retry_info}: {error_message}')
```

Comment:
Sort of unfortunate that ``retry_info`` has to remember to put spaces, but I do not think there is any way around it if we want to put it right between operation and the colon.